csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -8,7 +8,7 @@
     <VersionPrefix>3.21.9</VersionPrefix>
     <LangVersion>10.0</LangVersion>
     <Authors>Google Inc.</Authors>
-    <TargetFrameworks>netstandard1.1;netstandard2.0;net45;net50</TargetFrameworks>
+    <TargetFrameworks>netstandard1.1;netstandard2.0;net45;net50;net60</TargetFrameworks>
```

Comment:
I assume we genuinely *need* this for AOT support? (If we could continue to use the net50 target, that would avoid the package getting bigger.) I do expect to remove net50 at some point in 2023 though (I believe that we'd still have compatibility - just without some optimizations. Let me know if your understanding is different.)